js/src/options.ts

```diff
@@ -21,6 +21,7 @@ export type Options = {
     disableGrammar?: boolean;       // Option to disable Grammar
     disableTranslation?: boolean;   // Option to disable Translation
     disableLanguageDetection?: boolean;      // Option to disable Language Detection and use only the language that is explicitly specified on the chunk
+    providePlayPauseControl?: boolean;      // Option to prevent Play and Pause methods to be available.
```

Comment:
not prevent

// Option to provide Play and Pause methods in the launch response

Reply:
done